src/styles/index.scss

```diff
@@ -4,0 +1,7 @@
+@import '../styles/utils/variables';
+@import '../styles/blocks/general';
+@import '../styles/blocks/main';
+@import '../styles/blocks/header';
+@import '../styles/blocks/nav';
+@import '../styles/blocks/card';
+@import '../styles/blocks/stars';
```

Comment:
Verify that all imported block files exist and the paths are correct. This ensures that all styles are applied as expected.